src/huggingface_hub/hub_mixin.py

```diff
@@ -58,12 +58,12 @@ def save_pretrained(
                 json.dump(config, f)
 
         # saving model weights/files
-        self._save_pretrained(save_directory)
+        self._save_pretrained(save_directory, **kwargs)
```

Comment:
The `kwargs` are advertised as the kwargs to be passed to `push_to_hub` yet they're passed to `_save_pretrained`. I'd argue the kwargs to pass to each method ought to be different!